### DISCUSSION TITLE
Short syntax for nameof

### DISCUSSION THREAD

**Discussion by VardanKosakyan** [dotnet/csharplang#5503] [Language Ideas]
**Summary**

_nameof_ becomes more popular it would be nice to have short syntax for it. The possible syntax could be   '$$' preceding  symbol name.

**Example**

 Mapper.map( $$Model.Name, $$DTO.Name );
 
 instead of 
 
 Mapper.map( nameof(Model.Name), nameof(DTO.Name) );

**Comment by theunrepentantgeek** [comment]
Is this really needed? The **nameof** operator is clear and easy to read - it's syntax that doesn't really need to be learned. Making the language cryptic doesn't seem useful to me.

**Reply by VardanKosakyan** [reply]
nameof used very often, especially in such cases as mapper configuration, where it can be repeated multiple times per row and interferes with intuitive perception. Developer prefers characters over repeated words, that why,  i think,  C like languages are more popular than Visual Basic with a lot of  begin / end.

**Comment by HaloFour** [comment]
The language optimizes for clarity when reading the code.  The number of characters you have to type matters a lot less, especially when autocomplete will type most of them for you.

**Comment by Xyncgas** [comment]
```
//when you can't use "
dynamic a;
Console.WriteLine(nameof(a.HelloWorld));
```
As you can see, nameof is an important part of people's life
